plugins/zookeeper.yaml

```diff
@@ -3,11 +3,12 @@ version: 0.0.2
 title: Apache Zookeeper
 description: Log parser for Apache Zookeeper
 parameters:
-  - name: file_path
-    label: Zookeeper logs path
+  - name: file_log_path
     description: The absolute path to the Zookeeper logs
-    type: string
-    default: "/home/kafka/kafka/logs/zookeeper.log"
+    type: strings
```

Comment:
I like the idea of using strings here, but it does break compatibility. I am okay with this.

Reply:
👍